### ISSUE TITLE
JetBrains Fleet Plugin

### ISSUE THREAD

**Issue by rjohnson318** [rubberduck-ai/rubberduck-vscode#12]
Not sure if this is the right place to do this but I would like to request this be ported over to JetBrains Fleet as well. This is by far the best ChatGPT plugin and should be made the standard!

https://plugins.jetbrains.com/search?products=androidstudio&products=appcode&products=aqua&products=clion&products=dataspell&products=dbe&products=go&products=idea&products=idea_ce&products=idea_edu&products=mps&products=phpstorm&products=pycharm&products=pycharm_ce&products=pycharm_edu&products=rider&products=ruby&products=webstorm&search=gpt

**Comment by lgrammel** [comment]
Thanks! Great suggestion, this is something we want to do!

**Comment by lgrammel** [comment]
I totally agree. Sadly developing Rubberduck for a  different IDE would most likely be a completely separate effort though. I don't have time to work on it, but if someone wants to attempt writing an extension for Visual Studio, I can create another repository under the rubberduck-ai org. 

Ideally all Rubberduck extensions (for different IDEs) would consume the same Rubberduck conversation templates so they can be shared by project members.
